dt-core/global-functions.php

```diff
@@ -905,6 +905,97 @@ function dt_validate_date( string $date, string $format = 'Y-m-d H:i:s' ): bool
         }
     }
 
+    /**
+     * Dump and die
+     */
+    if ( !function_exists( 'dd' ) ) {
+        function dd( ...$params ) {
+            foreach ( $params as $param ) {
+                var_dump( $param );
+            }
+
+            exit;
+        }
+    }
+
+    if ( !function_exists( 'dt_sanitize_array' ) ) {
```

Comment:
This seems like a duplicate of `dt_recursive_sanitize_array`

Reply:
Removed in favor of `dt_recursive_sanitize_array`